spec/policies/grant_policy_spec.rb

```diff
@@ -117,7 +117,7 @@
 
     context 'with user not having a role on the grant' do
       context 'system_admin user' do
-        let(:user) { create(:system_admin_user) }
+        let(:user) { create(:system_admin_saml_user) }
```

Comment:
I think renamig user to saml user everywhere makes test little confusing. does the registered user have similar access? if type of user is only matters for authentication and not for functionality, I would keep user as it is.